packages/p2p/src/utils/payment-method.js

```diff
@@ -0,0 +1,6 @@
+export const getPaymentMethodIcon = payment_method_name => {
```

Comment:
Not related to this PR, Just to give you guys an idea 👀 

You guys can create a `useP2PAdvertiserPaymentMethods` hook using a combination of `useFetch` and `useRequest` hooks, Where the hook gives a list of advertiser payment methods as a state and method for `create`, `delete`, and `update`. The hook would also do the data transformation like sorting or grouping or adding missing information from API response to the entity, The icon for example or transforming the response into a list which as far as I remember it wasn't back then.

The endpoint is not subscribe-able but you can use `useInvalidateQuery` when CRUD operation happens, So eventually no need for any stores and the UI can only rely on the API response to render things.

```ts
const { data: advertiser_payment_methods, create, delete, update } = useP2PAdvertiserPaymentMethods();

// advertiser_payment_methods[0].icon;
// delete(1);
// ...
```

You can refer to `useWalletMigration` [here](https://github.com/binary-com/deriv-app/blob/1243f6d18d59a3ec0b5e7ebf2357b24543d43375/packages/hooks/src/useWalletMigration.ts) as an example of how to achieve these kinds of hook, And if any questions just ping me 🙇🏻 

> Psst, We have done this a long time ago in the [mobile codebase](https://github.com/regentmarkets/flutter-otc-cashier/blob/dev/lib/features/payment_methods/presentation/pages/advertiser_payment_methods_page.dart) and it works 👀 

cc: @markwylde-deriv @farrah-deriv @niloofar-deriv

Reply:
Thanks Farzin for your suggestion :pray:

Reply:
Thanks Farzin! 💯  Maybe we need to create a card for this.

Reply:
Thanks @farzin-deriv! We'll create a card for this.